App.tsx

```diff
@@ -492,21 +493,14 @@ const BaseApp: React.FunctionComponent<{
   });
 
   const linking = {
-    prefixes: [AvalancheCenterWebsites['NWAC'] + '/observations/#/view/'],
-    config: {
-      screens: {
-        Observations: {
-          screens: {
-            observation: 'observations/:id',
-          },
-        },
-      },
-    },
+    prefixes: [AvalancheCenterWebsites['NWAC']],
     getStateFromPath: (path: string, opts: {initialRouteName?: string; screens: PathConfigMap<object>} | undefined) => {
       if (initialUrl) {
-        // this url contains the whole url, like so: https://nwac.us/observations/#/observations/fb5bb19a-2b89-4c9c-91d2-eb673c5ab877
-        const url = new URL(initialUrl);
-        return getStateFromPath(path + '?share=true&share_url=' + url.origin + '/', opts);
```

Comment:
the back button navigation depends on this - so if you are going to fully take this out, that logic will need to change

Reply:
Oh yeah I am hoping we don't go that route ... :crossed_fingers: